modules/ROOT/pages/anypoint-mq/3.x/anypoint-mq-connector.adoc

```diff
@@ -0,0 +1,115 @@
+= Anypoint MQ Connector - 3.x
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Support Category: Select
+
+The Anypoint MQ connector provides publish-subscribe messaging to Mule apps using the Anypoint MQ cloud service.
+
+Anypoint MQ supports a wide range of messaging use cases: enterprise messaging patterns, asynchronous communications between microservices, IoT, and more. Anypoint MQ also has a REST API for publishing or consuming messages using any framework such as node.js, Java, Go, and devops bash scripts.
+
+For more information, see the xref:mq::index.adoc[Anypoint MQ documentation set] and additional resources listed in <<See Also>>.
+
+== Prerequisites
+
+To use this connector, you should be familiar with Anypoint Studio, Anypoint connectors, and Mule flows. The Anypoint MQ connector is free, but connecting to the service in Anypoint Platform requires a license.
+
+=== Compatibility
+
+[%header%autowidth.spread]
+|===
+|Software |Version
+|Mule Runtime |4.1.1 and later
+|Anypoint Studio |v7 and later
+|===
+
+IMPORTANT: This documentation applies to Mule 4 only. For Mule 3 compatible connector information, see xref:3.9@mule-runtime::anypoint-mq-connector.adoc[Mule 3 MQ Connector].
+
+To migrate from the Mule 4 connector version 2.x to the Mule 4 connector version 3.x, see the xref:anypoint-mq/3.x/anypoint-mq-new-major.adoc[Anypoint MQ 3.x Connector Migration Guide].
+
+== Install Anypoint MQ Connector
+
+In Anypoint Studio:
+
+. Create or open a Mule project.
+. Click the Exchange *(X)* icon in the Studio task bar.
+. Enter your Anypoint Platform username and password and click *Sign in*.
+. Select *All assets* and search for "MQ"
+. Select "Anypoint MQ Connector - Mule 4", and click *Add to project*.
+. Follow the prompts to install the connector.
+
+For standalone apps or apps written in XML:
+
+. Open your Mule project in Anypoint Studio.
+. Add the connector as a dependency in the pom.xml file:
++
+[source,xml,linenums]
+----
+<dependency>
+  <groupId>com.mulesoft.connectors</groupId>
+  <artifactId>anypoint-mq-connector</artifactId>
+  <version>${RELEASE}</version>
+  <classifier>mule-plugin</classifier>
+</dependency>
+----
+
+For the latest POM, Gradle, or SBT dependencies, see the  https://www.anypoint.mulesoft.com/exchange/com.mulesoft.connectors/anypoint-mq-connector/[Exchange Anypoint MQ Connector] asset. Click *Dependency Snippets* for a complete list.
```

Comment:
Same as in 2.x -- If you use RELEASE, which is the best, you don't need "For the latest" -- the Exchange link is only for users who want to specify a version (not sure why). Also Gradle and SBT have no MQ context.